### DISCUSSION TITLE
ARAM Same Hero Mode should not count towards winrate or MMR

### DISCUSSION THREAD

**Discussion by Will-Crain** [Heroes-Profile/heroesprofile#1058] [Ideas]
I play a lot of ARAM, and I've been noticing that my team almost always has someone leave in hero select once they realize it's a Same Hero match, or someone who purposefully AFKs or feeds the enemy team to make the match end faster. I suspect this is also happening on enemy teams.

In theory, keeping this game mode lumped in with normal ARAM matches seems fine. However, given many player's reactions to Same Hero mode and the destructive behavior that follows, I believe these matches _should not_ mutate a player's stats.

Is this possible to implement, or is it even something others care for?

**Comment by Zemill** [comment]
The purpose of the site is to attempt to try and accurately match your in game stats.  These types of behaviors still show up on your in game profile.  Even the people who leave.

I would say it doesnt make sense to change that, let me know your thoughts
